spec.bs

```diff
@@ -164,6 +165,7 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     urlPrefix: interaction.html
       text: activation notification; url: activation-notification
       text: consume user activation; url: consume-user-activation
+      text: transient activation; url: transient-activation 
```

Comment:
This is exported by HTML so shouldn't need to include this here. The same seems to be true for fully active.

I believe if [it appears in webdex](https://dontcallmedom.github.io/webdex/t.html#Transient%20activation%40%40html%25%25dfn), it is exported and doesn't need to be added to this section manually. (Also for transient activation I checked the HTML source).

Reply:
Got it, removed them both. Thanks for the webdex link! Was wondering where all the autodef magic was coming from.